source/slang/slang-ast-support-types.h

```diff
@@ -1285,6 +1285,7 @@ namespace Slang
     struct WitnessTable : RefObject
     {
         RequirementDictionary requirementDictionary;
+        Type* baseType;
```

Comment:
This probably merits a comment. I assume that `baseType` represents the type that the witness table witnesses conformance *to* (currently an interface), and not the type it witnesses conformance *of*.